src/auth/auth-token-integration.spec.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from './auth-cross-domain-iframe';
 
 import 'jasmine-ajax';
-
+import { BBAuthGetDomain } from './auth-get-domain';
```

Comment:
Move this above the previous import and make it multi-line to match the other imports in this file.